settings.py

```diff
@@ -0,0 +1,16 @@
+"""
+settings module for the app
+"""
+
+FIRM_ID = 111
+PUBLICATION_ID = 2
+
+
+def get_url_firm():
```

Comment:
that's a wrapper function that's returning the whole url instead defining it in apps main file